_data/people/fkw888.yml

```diff
@@ -6,6 +6,13 @@ institution: University of California, San Diego
 website: https://www-physics.ucsd.edu/fac_staff/fac_profile/faculty_description.php?person_id=494
 photo: /assets/images/team/Frank-Wuerthwein.jpg
 presentations:
+  - title: "State of OSG"
+    date: 19 Mar 2019
+    url: https://indico.cern.ch/event/759388/contributions/3353296/attachments/1814351/2964744/osgahm19-20190319-am11-fkw-a.pdf
+    meeting: "HOW2019 (Joint HSF/OSG/WLCG Workshop)"
+    meetingurl: https://indico.cern.ch/event/759388/
+    location: Jefferson Lab
+    focusarea: osglhc
```

Comment:
The proper tag is "osg" for the IRIS-HEP website. Since we have only one bit of the OSG in here...

Reply:
OK, I will change it. But note that I was following the instructions here: <http://iris-hep.org/docs/newteammember>. It lists `osglhc` as the only OSG option for `focusarea`.

Reply:
Crap! You are right. I was basing it off some internal notes. I'd not realized that had been updated. Ok - we have to keep it osglhc then. Sorry!

Reply:
OK, reverted.

Reply:
Great!